Content.Shared/Devour/SharedDevourSystem.cs

```diff
@@ -47,6 +48,14 @@ protected void OnDevourAction(EntityUid uid, DevourerComponent component, Devour
         args.Handled = true;
         var target = args.Target;
 
+        // SS220 Zombie and Infected check before devour begin
+        if (args.Handled || component.Blacklist?.IsValid(args.Target, EntityManager) == true)
```

Comment:
```suggestion
        if (args.Handled || component.Blacklist?.IsValid(args.Target, EntityManager))
```

Методы, возвращающие `bool`, можно не приравнивать к какому-либо значению.